demisto_sdk/commands/common/hook_validations/incident_field.py

```diff
@@ -301,3 +303,21 @@ def is_cliname_is_builtin_key(self):
                 f"{self.file_path}: cliName field can not be {cliname} as it's a builtin key."
             )
         return is_valid
+
+    def is_valid_from_version(self):
+        error_msg = None
+        try:
+            server_base_version = self.current_file.get('fromVersion').split('.')[0]
+            if server_base_version and int(server_base_version) >= 5:
+                return True
+            else:
+                error_msg = f'{self.file_path}: "fromVersion" mast be at least 5.0.0'
+        except AttributeError:
+            pass
+        except ValueError:
+            pass
+
+        if not error_msg:
```

Comment:
Please handle the error message in the expect and print the error catched.
also, please use only 1 return in the function, and return a true/false value according to the flow.